### PR TITLE
Add name to user response

### PR DESCRIPTION
**Pull request by pezholio**
We also need the name of the user who created an application, so adding this in

### CODE REVIEW

**Comment by pezholio** [comment]
[Confirm if there's enough information in assessment](https://trello.com/c/Jd5xAwWR/873-confirm-if-theres-enough-information-in-assessment)

File: src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt [hunk 1]
```diff
@@ -39,7 +39,10 @@ class UserService(
       is ClientResult.Failure -> staffUserDetailsResponse.throwException()
     }
 
-    if (deliusUser.email !== user.email || deliusUser.telephoneNumber !== user.telephoneNumber) {
+    val name = "${deliusUser.staff.forenames} ${deliusUser.staff.surname}"
+
+    if (deliusUser.email !== user.email || deliusUser.telephoneNumber !== user.telephoneNumber || name != user.name) {
```
Comment (edavey) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt:44]: Would the purpose of this perhaps be clearer if this became a predicate function, e.g. `hasUserChanged()`? I think that's what this is doing: updating our user record if we detect a change at Delius?

**Review by edavey — APPROVED** [review]
👍

File: src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt (hunk 1 above)
Comment (pezholio) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt]: Good idea! I've made that change 👍

**Pull request merged**